Src/CSharpier/Doc.cs

```diff
@@ -92,6 +92,7 @@ public class LeadingComment : Doc
     {
         public CommentType Type { get; set; }
         public string Comment { get; set; } = string.Empty;
+        public bool AddExtraIndent { get; set; }
```

Comment:
For some reason I didn't think to try just using Indent. Using it did require some changes but this keeps the core code much cleaner.
I also played around with LeadingComment, and it looks like it can probably go away. With the changes I made https://github.com/belav/csharpier/issues/22 it seems like it isn't necessary. It wasn't completely straightforward to remove them, so I created a new issue for it - https://github.com/belav/csharpier/issues/67